builtin/credential/aws/pkcs7/ber.go

```diff
@@ -1,3 +1,6 @@
+// Copyright (c) HashiCorp, Inc.
```

Comment:
This is a great catch! It appears the copywrite config entry has `credentials` (plural) instead of `credential` in the path:

```
- builtin/credentials/aws/pkcs7/**
+ builtin/credential/aws/pkcs7/**
```

Reply:
Great catch @marcboudreau!  I have addressed this comment in the new [PR](https://github.com/hashicorp/vault/pull/19379).